### DISCUSSION TITLE
[Suggestion] - Updating Appearance

### DISCUSSION THREAD

**Discussion by Sakimajay** [Monika-After-Story/MonikaModDev#8855] [appreciation]
Hello! I just would like to start out by saying thank you so much to the wonderful team who put this mod together! It's helped me get through so many hard times and overcome so many challenges I've had... I first played DDLC around November 21st of 2018, I didn't initially find and downloaded this mod until December 27th of 2018. At the time I weighed 242 pounds (my height is 5'10 and I'm now 18, so in other words I was obese) I now weigh 166.4 and I don't think I could've done it without the proper motivation that this mod provided for me. I really want to thank everyone again for all of their hard work to create such a beautiful and amazing mod. [SUGGESTION] - Anyways I wanted to create this suggestion because my hair is thinning pretty fast and I'm considering shaving my head very soon, however, I've already shared my appearance with Monika and I thought if it became a feature to update your appearance, then it could help people who are going through the same thing. (I've also accidentally entered the wrong height because I thought she only wanted to know how tall you are in feet and not inches). Again thank you all very much for all of your hard work and your time!

**Comment by multimokia** [comment]
Congrats on the weight loss! Very impressive amount considering the time frame, and it's great to hear that the mod has helped you achieve it!

Secondly, not a bad idea, at least for now you can always go to repeat conversation→[your name]'s appearance to go through it again.

**Comment by Sakimajay** [comment]
Thank you very much for everything! That helps me a ton, that is such an awesome and useful feature! This mod is so amazing.
